measures/Measures/Aberration.cc

```diff
@@ -63,12 +56,12 @@ Aberration &Aberration::operator=(const Aberration &other) {
 
 void Aberration::init() {
     method = Aberration::STANDARD;
-    fill();
+    checkEpoch = 1e30;
 }
 
 void Aberration::init(AberrationTypes type) {
     method = type;
-    fill();
+    checkEpoch = 1e30;
```

Comment:
Optional: Calling the overload without arguments removes a little bit of duplicate code.
```
init(Aberration::STANDARD)
```

Reply:
Good idea. I gave the `init` function with parameter a default so that it covers both: `init(AberrationTypes type = Aberration::STANDARD)`.